### ISSUE TITLE
sudo powers

### ISSUE THREAD

**Issue by fjbriones** [paulinosalmon/CoE135Project#1]
Do I need sudo powers for this to work? Can you think of any way to do this without using sudo.

**Comment by paulinosalmon** [comment]
Yes, sir. Sudo/root is required for python3 live_parser.py as the default logs are created under /debug which isn't accessible to the normal user to upload easily using Google Drive API. One way to eliminate the necessity of sudo is to change where the logs are written by default in keylogger.c, but we did not have much time to tweak with it so we decided to manipulate the log with python instead.
